docs/docs/tutorial.md

```diff
@@ -220,6 +221,7 @@ First, add the third-party library **marked** to your application. This is a Jav
 ```html{7}
```

Comment:
Need to bump this if you're adding a line so that the correct line stays highlighted.